layouts/partials/footer.html

```diff
@@ -37,6 +37,12 @@
             </a>
         {{ end }}
 
+        {{ if .Site.Params.instagram }}
+            <a href="https://www.instagram.com/{{ .Site.Params.linkedin }}" target="_blank">
```

Comment:
I think you mean `.Site.Params.instagram` (rather than linkedin). I've made this mistake before too.

Reply:
Otherwise, LGTM

Reply:
@mtn you’re right.. see, this is why we can’t have nice things. Thanks for fixing this error!